provers/celestia-prover/README.md

```diff
@@ -64,14 +64,11 @@ Push new image to GHCR:
 make publish-celestia-prover-docker
 ```
 
-### Regenerate ELFs
+## Benchmarks
 
-Currently the celestia-prover ELFs aren't automatically rebuilt so they must be manually rebuilt if you modify the SP1 source code.
+How long does it take to generate a proof?
 
-```shell
-# Regenerate mock-membership-elf
-~/.sp1/bin/cargo-prove prove build --elf-name mock-membership-elf --output-directory provers/celestia-prover/elf/ --packages mock-membership --no-default-features
-
-# Regenerate mock-update-client-elf
-$ ~/.sp1/bin/cargo-prove prove build --elf-name mock-update-client-elf --output-directory provers/celestia-prover/elf/ --packages mock-update-client --no-default-features
-```
+| Proof            | Time       | Cycles |
+|------------------|------------|--------|
+| State Transition | 80s - 235s | TBD    |
+| Membership       | 105s       | TBD    |
```

Comment:
👍 

Is the variance dependent on the validator set - are there any other factors that would contribute to generation time

Reply:
Discussed on Slack. I think the variance is based on which machine in the prover networks services the request b/c the demo has the same SimApp / validator set / and number of blocks each time I create a proof and there is variance.